### ISSUE TITLE
[BUG] filter out ISO images from the Harvester node driver UI

### ISSUE THREAD

**Issue by guangbochen** [harvester/harvester#809]
**Describe the bug**
Currently, only cloud image is supported by the Harvester node driver, so would be nice if the node driver UI can filter out those ISO images to avoid accidental selection.

**To Reproduce**
1. Go to the Rancher
2. Click to add cluster and select the `Harvester` node driver
3. create a node template and choose available images

**Expected behavior**
Filter out ISO image from the harvester node template UI.

**Environment:**
 - Harvester ISO version: harvester-v0.2.0-rc5
 - Installation Mode: ISO/app-mode
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630):

**Additional context**
![image](https://user-images.githubusercontent.com/4569037/116657037-53392100-a9c0-11eb-9f14-935e5de2491b.png)

**Comment by guangbochen** [comment]
You should be able to verify this by replacing the Harvester node driver on the [Rancher UI](https://docs.harvesterhci.io/v0.2/rancher/node-driver/#how-to-add-harvester-node-driver-in-dev-mode),
the master version of Harvester node driver and UI assets are available as below:
- https://harvester-node-driver.s3.amazonaws.com/ui/master/component.js
- https://harvester-node-driver.s3.amazonaws.com/driver/master/docker-machine-driver-harvester-amd64.tar.gz

**Comment by juadk** [comment]
First, I was able to reproduce the issue with the current Harvester driver node version. The ISO images appeared in the available images.

The fix worked well for me:
Mode: ISO
Version: master-3b6b8a6d-head
Different images were added .img & .iso
![image](https://user-images.githubusercontent.com/6025636/120811116-ee875e00-c54b-11eb-93f7-6805c335e850.png)

I updated the Harvester node driver with the master version.
![image](https://user-images.githubusercontent.com/6025636/120811263-137bd100-c54c-11eb-8296-cc505d90eb4c.png)

I can't see the ISO images anymore.
![image](https://user-images.githubusercontent.com/6025636/120811471-4a51e700-c54c-11eb-93d7-ac2ab6eff32b.png)
